Tests/GettableSpec.swift

```diff
@@ -16,6 +16,29 @@ extension Gettable {
     }
 }
 
+struct DateResponse: Decodable {
+    var date: Date
+}
+
+struct DateRequest: Gettable {
+    public typealias ResponseType = DateResponse
+    public var path: String { return "/test" }
+    public var dateDecodingStrategy: JSONDecoder.DateDecodingStrategy {
+        return .formatted(dateFormatter())
+    }
+}
+
+extension DateRequest {
+    func dateFormatter() -> DateFormatter {
+        let formatter = DateFormatter()
+        formatter.calendar = Calendar(identifier: .iso8601)
+        formatter.locale = Locale(identifier: "en_US_POSIX")
+        formatter.timeZone = TimeZone(secondsFromGMT: 0)
+        formatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSXXXXX"
+        return formatter
+    }
+}
+
```

Comment:
Move these test structs to the `Test Classes` (a horribly named directory and I apologize to all for that) directory and make a new file there which contains these.

Reply:
👍